TestFoundation/TestNSDictionary.swift

```diff
@@ -222,6 +223,12 @@ class TestNSDictionary : XCTestCase {
         }
     }
 
+    func test_settingWithStringKey() {
+        let dict = NSDictionary()
```

Comment:
Don't you mean `NSMutableDictionary()`? On macOS, the test code as given results in a compile-time error: `error: cannot assign through subscript: subscript is get-only`.

Reply:
Whoops. Thanks @xwu. Fixed and squashed.